package.json

```diff
@@ -124,32 +124,54 @@
           "light": "assets/document-search-light.svg",
           "dark": "assets/document-search-dark.svg"
         }
+      },
+      {
+        "command": "ocaml.open-repl",
+        "category": "OCaml",
+        "title": "Open REPL"
+      },
+      {
+        "command": "ocaml.evaluate-selection",
+        "category": "OCaml",
+        "title": "Evaluate Selection"
       }
     ],
     "keybindings": [
       {
         "command": "ocaml.switch-impl-intf",
         "key": "Alt+O",
-        "when": "editorLangId  == ocaml || editorLangId  == ocaml.interface || editorLangId == reason || editorLangId == ocaml.ocamllex || editorLangId == ocaml.menhir"
+        "when": "editorLangId == ocaml || editorLangId == ocaml.interface || editorLangId == reason || editorLangId == ocaml.ocamllex || editorLangId == ocaml.menhir"
       },
       {
         "command": "editor.action.codeAction",
         "key": "Alt+D",
         "args": {
           "kind": "destruct"
         },
-        "when": "editorLangId  == ocaml || editorLangId == reason"
+        "when": "editorLangId == ocaml || editorLangId == reason"
```

Comment:
I think this toplevel only works for OCaml unfortunately.

Reply:
That's for the destruct command, I just removed a space